docs/source/roles/researcher/troubleshooting.md

```diff
@@ -36,6 +36,18 @@ Either log into that workspace and close Firefox, or follow the instructions [he
 Your profile is likely stored under **~/snap/firefox/common/.mozilla/firefox**.
 :::
 
+## {{ zzz }} Idle screen lock
```

Comment:
```suggestion
## {{zzz}} Idle screen lock
```
I'm not sure the code that replaces this with `:zzz:` and then replaces that with an emoji will work correctly with spaces.

Reply:
The emoji did render locally, let's change it to avoid whitespace problems and stay consistent though :+1:.